### PR TITLE
CN-7122: Change wording interest

### PR DESCRIPTION
**Pull request by djalfirevic**
## Checklist (Mandatory):

- [x] I have named the PR correctly (Check PR1)
- [x] Check the commit’s or even all commits message styles matches our requested structure. (issue number, etc.) (Check CR10)
- [x] I have tested new code locally (Check CR3)
- [x] I have done a smoke test of the larger feature locally (Check CR3)
- [x] I have added at least two reviewers (Check PR5)
- [x] I have assigned the PR to myself (Check PR4)
- [x] I have merged develop (parent) branch into my feature branch and resolved all conflicts (Check PR7)

## Checklist (Optional):

- [ ] I have added necessary documentation, including instructions how to test (Check CR8)
- [ ] I have added tests that prove my fix/implementation is effective or that my feature works
- [ ] I have written the code according to the adopted style guide? (Check CR7)

## Additional notes

> What does this implement/fix? Explain your changes. (optional)
> Are there any scripts that need running (eg. yarn install)?
> Are there any edge cases that need checking?
> Are there any specific test instructions?

[MVPWorkshop Pull Request & Code Review Guidelines](https://docs.google.com/document/d/1TcuU1JBhlGZLtpy0osSNinXQ2piP_yrDuRcXdqaOS_U/edit#heading=h.5b6kpdr4btq8)

### CODE REVIEW

File: app/components/modals/InterestDueModal/InterestDueModal.js [hunk 1]
```diff
@@ -40,11 +40,11 @@ class InterestDueModal extends Component {
     return (
       <CelModal name={modalName}>
         <CelText type="H2" align="center" weight="bold">
-          Interest Payment
+          Rewards Payment
```
Comment (fjovakaric) [inline app/components/modals/InterestDueModal/InterestDueModal.js:43]: this is for loans

File: app/components/modals/InterestDueModal/InterestDueModal.js [hunk 2]
```diff
@@ -40,11 +40,11 @@ class InterestDueModal extends Component {
...
         </CelText>
 
         <CelText align="center" margin="10 0 10 0">
-          Your interest due is
+          Your rewards due is
```
Comment (fjovakaric) [inline app/components/modals/InterestDueModal/InterestDueModal.js:47]: this is for loans

File: app/components/modals/InterestDueModal/InterestDueModal.js [hunk 3]
```diff
@@ -63,7 +63,7 @@ class InterestDueModal extends Component {
               Payment Period
             </CelText>
             <CelText type="H7" weight="bold">
-              Monthly Interest
+              Monthly Rewards
```
Comment (fjovakaric) [inline app/components/modals/InterestDueModal/InterestDueModal.js:66]: this is for loans

File: app/components/modals/InterestDueModal/InterestDueModal.js [hunk 4]
```diff
@@ -109,7 +109,7 @@ class InterestDueModal extends Component {
               closeModal();
             }}
           >
-            Pay Interest
+            Pay Rewards
```
Comment (fjovakaric) [inline app/components/modals/InterestDueModal/InterestDueModal.js:112]: this is for loans

File: app/components/modals/InterestDueModal/InterestDueModal.stories.js [hunk 5]
```diff
@@ -14,7 +14,7 @@ const InterestDueModalStories = () => (
     <CelButton
       onPress={() => store.dispatch(openModal(MODALS.INTEREST_DUE_MODAL))}
     >
-      Open Interest Due modal
+      Open Rewards Due modal
```
Comment (fjovakaric) [inline app/components/modals/InterestDueModal/InterestDueModal.stories.js:17]: this is for loans

File: app/components/modals/PrepayDollarInterestModal/PrepayDollarInterestModal.js [hunk 6]
```diff
@@ -7,9 +7,9 @@ const PrepayDollarInterestModal = props => {
   return (
     <InfoModal
       name={MODALS.PREPAY_DOLLAR_INTEREST_MODAL}
-      heading={"Prepay Your Interest in Dollars"}
+      heading={"Prepay Your Rewards in Dollars"}
```
Comment (fjovakaric) [inline app/components/modals/PrepayDollarInterestModal/PrepayDollarInterestModal.js:10]: this is for loans

File: app/components/modals/PrepayDollarInterestModal/PrepayDollarInterestModal.js [hunk 7]
```diff
@@ -7,9 +7,9 @@ const PrepayDollarInterestModal = props => {
...
       paragraphs={[
-        "You can prepay your interest in dollars with a wire transfer from your bank. Payment details for a wire transfer will be provided on the next screen. For more information on how to initiate a wire transfer, you will need to contact your bank or local branch.",
+        "You can prepay your rewards in dollars with a wire transfer from your bank. Payment details for a wire transfer will be provided on the next screen. For more information on how to initiate a wire transfer, you will need to contact your bank or local branch.",
```
Comment (fjovakaric) [inline app/components/modals/PrepayDollarInterestModal/PrepayDollarInterestModal.js:12]: this is for loans

File: app/components/modals/PrepaymentSuccessfulModal/PrepaymentSuccessfulModal.js [hunk 8]
```diff
@@ -41,11 +41,11 @@ class PrepaymentSuccessfulModal extends Component {
           margin={"0 20 5 20"}
           weight={"700"}
         >
-          {` Successfully Prepayed ${formData.prepaidPeriod} Months of Interest`}
+          {` Successfully Prepayed ${formData.prepaidPeriod} Months of Rewards`}
```
Comment (fjovakaric) [inline app/components/modals/PrepaymentSuccessfulModal/PrepaymentSuccessfulModal.js:44]: this is for loans

File: app/components/modals/PrepaymentSuccessfulModal/PrepaymentSuccessfulModal.js [hunk 9]
```diff
@@ -41,11 +41,11 @@ class PrepaymentSuccessfulModal extends Component {
...
         </CelText>
         <CelText align={"center"} margin={"5 20 20 20"}>
           {`This means you don’t need to make any payments in the next ${formData.prepaidPeriod} months!
-          We will let you know when your interest payment resumes.`}
+          We will let you know when your rewards payment resumes.`}
```
Comment (fjovakaric) [inline app/components/modals/PrepaymentSuccessfulModal/PrepaymentSuccessfulModal.js:48]: this is for loans

File: app/components/modals/ReferralSendModal/ReferralSendModal.js [hunk 10]
```diff
@@ -120,7 +120,7 @@ class ReferralSendModal extends Component {
         <CelText type="H6" align={"left"} style={style.explanation}>
           *$20 in BTC is distributed when a new user makes a first deposit of
           $200 or more using your referral code at the time of signup. BTC
-          reward will be unlocked 30 days after the initial deposit.
+          rewards will be unlocked 30 days after the initial deposit.
```
Comment (fjovakaric) [inline app/components/modals/ReferralSendModal/ReferralSendModal.js:123]: should be reward

File: app/components/modals/RegisterPromoCodeModal/RegisterPromoCodeModal.js [hunk 11]
```diff
@@ -261,11 +261,11 @@ class RegisterPromoCodeModal extends Component {
       code.coin
     } when you deposit $${code.minimumAmount} or more within the next ${
       code.maximumDays
-    } days. Your reward will be locked in your wallet for 30 days. If you withdraw $${
+    } days. Your rewards will be locked in your wallet for 30 days. If you withdraw $${
```
Comment (fjovakaric) [inline app/components/modals/RegisterPromoCodeModal/RegisterPromoCodeModal.js:264]: should stay reward

File: app/components/modals/RegisterPromoCodeModal/RegisterPromoCodeModal.js [hunk 12]
```diff
@@ -261,11 +261,11 @@ class RegisterPromoCodeModal extends Component {
...
       code.minimumAmount
     } or more from your wallet before ${moment()
       .add(30, "days")
-      .format("DD/MM/YY")}, your reward will be canceled.`;
+      .format("DD/MM/YY")}, your rewards will be canceled.`;
```
Comment (fjovakaric) [inline app/components/modals/RegisterPromoCodeModal/RegisterPromoCodeModal.js:268]: should stay reward

File: app/components/modals/WithdrawalInfoModal/WithdrawalInfoModal.js [hunk 13]
```diff
@@ -29,7 +29,7 @@ class WithdrawalInfoModal extends Component {
       case "CEL":
         title = "Are you sure you want to withdraw CEL? ";
         body =
-          "The longer you HODL and the more you HODL, the more interest you'll earn with Celsius. Withdrawing your funds will reduce the amount of interest you could potentially earn.";
+          "The longer you HODL and the more you HODL, the more rewards you'll earn with Celsius. Withdrawing your funds will reduce the amount of you could potentially earn.";
```
Comment (fjovakaric) [inline app/components/modals/WithdrawalInfoModal/WithdrawalInfoModal.js:32]: missing a word

File: app/components/molecules/PaymentCard/PaymentCard.js [hunk 14]
```diff
@@ -43,7 +43,7 @@ class PaymentCard extends Component {
       if (reason) {
         actions.showMessage(
           "success",
-          "You have successfully changed interest payment method"
+          "You have successfully changed rewards payment method"
```
Comment (fjovakaric) [inline app/components/molecules/PaymentCard/PaymentCard.js:46]: should stay interest

File: app/components/molecules/TxReferralSection/TxReferralSection.js [hunk 15]
```diff
@@ -79,7 +79,7 @@ class TxReferralSection extends Component {
                 color={getColor(COLOR_KEYS.HEADLINE)}
                 weight="600"
               >{` ${moment(transaction.time).format("D MMM YYYY")} `}</CelText>
-              , your referral reward will unlock.
+              , your referral rewards will unlock.
```
Comment (fjovakaric) [inline app/components/molecules/TxReferralSection/TxReferralSection.js:82]: should stay reward

File: app/components/screens/PaymentCel/PaymentCel.js [hunk 16]
```diff
@@ -116,14 +116,14 @@ class PaymentCel extends Component {
             >
               Loyalty Level
             </CelText>
-            , your next interest payment would be:
+            , your next rewards payment would be:
```
Comment (fjovakaric) [inline app/components/screens/PaymentCel/PaymentCel.js:119]: should stay interest

File: app/components/screens/PaymentCel/PaymentCel.js [hunk 17]
```diff
@@ -116,14 +116,14 @@ class PaymentCel extends Component {
...
           </CelText>
           <CelText type={"H1"} weight={"700"} align={"center"}>
             {celDiscount} less
           </CelText>
           <Separator margin={"10 0 12 0"} />
           <CelText>
-            Decrease your monthly interest payment from{" "}
+            Decrease your monthly rewards payment from{" "}
```
Comment (fjovakaric) [inline app/components/screens/PaymentCel/PaymentCel.js:126]: should stay interest

File: app/components/screens/PrincipalPayment/PrincipalPayment.js [hunk 18]
```diff
@@ -134,7 +134,7 @@ class PrincipalPayment extends Component {
         {payOutPrincipalFromCollateral ? (
           <Card color={STYLES.COLORS.ORANGE}>
             <CelText weight="300" alignItems="center" color="#FFFFFF">
-              Your principal loan amount and any unpaid interest will
+              Your principal loan amount and any unpaid rewards will
```
Comment (fjovakaric) [inline app/components/screens/PrincipalPayment/PrincipalPayment.js:137]: should stay interest

File: app/redux/user/userDataActions.js [hunk 19]
```diff
@@ -178,7 +178,7 @@ function setUserAppSettings(data) {
               return dispatch(
                 showMessage(
                   "success",
-                  `Congrats! You have chosen to earn interest income in CEL for all deposited coins. Interest has already been calculated for this week, so you will receive interest in CEL beginning Monday, ${currentDate
+                  `Congrats! You have chosen to earn rewards income in CEL for all deposited coins. A rewards has already been calculated for this week, so you will receive interest in CEL beginning Monday, ${currentDate
```
Comment (fjovakaric) [inline app/redux/user/userDataActions.js:181]: `A reward` not `A rewards`

File: app/redux/user/userDataActions.js [hunk 20]
```diff
@@ -188,7 +188,7 @@ function setUserAppSettings(data) {
               return dispatch(
                 showMessage(
                   "success",
-                  `You have chosen to earn interest income in-kind for all deposited coins. Interest has already been calculated for this week, so you will receive interest in-kind beginning Monday, ${currentDate
+                  `You have chosen to earn rewards income in-kind for all deposited coins. A rewards has already been calculated for this week, so you will receive interest in-kind beginning Monday, ${currentDate
```
Comment (fjovakaric) [inline app/redux/user/userDataActions.js:191]: same as above

File: app/components/organisms/WalletDetailsCard/WalletDetailsCard.js [hunk 21]
```diff
@@ -88,7 +88,7 @@ class WalletDetailsCard extends PureComponent {
             <TouchableOpacity
               onPress={() => actions.navigateTo("InterestRates")}
             >
-              <CelText link>Weekly Reward Rates</CelText>
+              <CelText link>Weekly Rewards Rates</CelText>
```
Comment (fjovakaric) [inline app/components/organisms/WalletDetailsCard/WalletDetailsCard.js:91]: check everywhere for rewards rates, should be singular

File: app/components/organisms/CommunityDashboard/CommunityDashboard.js [hunk 22]
```diff
@@ -109,7 +109,7 @@ class CommunityDashboard extends Component {
       number = formatter.usd(communityStats.average_interest_earned_usd);
     }
     if (name === "REWARDS" && button === "Rates") {
-      explanationText = "Interest rates";
+      explanationText = "Rewards rates";
```
Comment (fjovakaric) [inline app/components/organisms/CommunityDashboard/CommunityDashboard.js:112]: reward rates

**Review by fjovakaric — CHANGES_REQUESTED** [review]
